lib/ain-evm/src/block.rs

```diff
@@ -189,11 +215,14 @@ impl BlockService {
                 let gas_ratio = if block.header.gas_limit == U256::zero() {
                     f64::default() // empty block
                 } else {
-                    block.header.gas_used.as_u64() as f64 / block.header.gas_limit.as_u64() as f64
+                    u64::try_from(block.header.gas_used)? as f64
+                        / u64::try_from(block.header.gas_limit)? as f64
                 };
 
-                (base_fee, gas_ratio)
+                Ok((base_fee, gas_ratio))
             })
+            .collect::<Result<Vec<(U256, f64)>>>()?
+            .into_iter()
             .unzip();
```

Comment:
Could `fold` instead of collect -> into_iter -> unzip

Reply:
Fixed: 88e528c040425f08134ca14bf731561f55a3234f